### PR TITLE
[hi]Localize content/en/docs/reference/glossary/app-container.md

### LINKED ISSUE #35941: [hi] Localize /en/docs/reference/glossary/app-container.md
[hi] Localize /en/docs/reference/glossary/app-container.md
# Instructions

- Review a few [examples of localized glossary entries](https://github.com/kubernetes/website/issues/35600#issuecomment-1214215859).
- In the examples, notice which elements are translated and which ones are not.

## Translating the frontmatter

In the frontmatter, that is, between the `---` markers that enclose the first dozen lines:
- Translate the `title` and `short_description` values.
- Do not translate the `id`, `aka`, `tags` values. We will systematically translate those later on in a separate project.

## Translating the content body

Below the frontmatter, in the content body:
- Translate almost everything.
- If you encounter a glossary shortcode, for example, `{{< glossary_tooltip text="containers" term_id="container" >}}`, translate the `text` value. But do not translate the `term_id` value.

# Procedure:
1. In this issue, assign this issue to yourself by commenting: `/assign <your_github_id>`
1. Open PR #35942.
1. In the PR, select all the lines in the file and click `+` to comment. Screenshot 1
1. In the comment, click the **+/-** suggestion icon (screenshot 2). Screenshot 2
1. In the suggestion, localize the content to Hindi.
1. IMPORTANT: Click the **Start a review** button.
1. IMPORTANT: Click **Review changes**, **Comment**, and **Submit review**. Screenshot 3

*Note*: By completing these steps, your GitHub profile will give you credit for performing a code review!

As follow-up steps, I will do the following:
- Apply your suggested localization to the PR.
- Ask a reviewer to review the localized content.
- Update the content as needed.
- Get the PR approved and merged.

This feature issue is part of the following project:
- https://github.com/kubernetes/website/issues/35600

### PR DESCRIPTION
**Pull request by taniaduggal**
Fixes #35941

### CODE REVIEW

File: content/hi/docs/reference/glossary/app-container.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: ऐप कंटेनर (App Container)
+id: app-container
+date: 2019-02-12
+full_link:
+short_description: >
+  एक कंटेनर एक कार्यभार का हिस्सा चलाने के लिए प्रयोग किया जाता है। प्रारंभिक कंटेनर की तुलना में।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/app-container.md:7]: I feel  कार्यभार is a literal machine translation & we shouldn't be using it.

File: content/hi/docs/reference/glossary/app-container.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+aka:
+tags:
+- workload
+---
+एप्लिकेशन {{<glossary_tooltip text="पात्र" term_id="container" >}} (या ऐप कंटेनर) {{<glossary_tooltip text="पॉड" term_id="pod" >}} में, {{<glossary_tooltip text ="इनिट कंटेनर" term_id="init-container" >}} स्टार्टअप पूरा होने के बाद शुरू होता है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/app-container.md:13]: Why have we used पात्र as the text here when we're using container in its machine translated form everywhere else?

File: content/hi/docs/reference/glossary/app-container.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+<!--more-->
+
+एक  इनिट कंटेनर के साथ, संपूर्ण
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/app-container.md:17]:
```suggestion
एक इनिट कंटेनर के साथ, संपूर्ण
```

File: content/hi/docs/reference/glossary/app-container.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+आप {{< glossary_tooltip text="कार्यभार" term_id="workload" >}} के लिए महत्वपूर्ण आरंभीकरण विवरण अलग कर सकते हैं।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/app-container.md:18]: Still not sure about कार्यभार and I presume you meant app workload 

```suggestion
ऐप {{< glossary_tooltip text="कार्यभार" term_id="workload" >}} के लिए महत्वपूर्ण आरंभीकरण विवरण अलग कर सकते हैं।
```

Comment (divya-mohan0209) [reply content/hi/docs/reference/glossary/app-container.md:18]: If it was intended to mean you, in a respectful way, then the structure of this sentence needs to be modified as below.

एक इनिट कंटेनर के साथ, आप संपूर्ण  {{< glossary_tooltip text="कार्यभार" term_id="workload" >}} के लिए महत्वपूर्ण आरंभीकरण विवरण अलग कर सकते हैं।

**Review by divya-mohan0209 — COMMENTED** [review]
Few suggestions, hope you find them useful.

File: content/hi/docs/reference/glossary/app-container.md (hunk 1 above)
Comment (taniaduggal) [reply content/hi/docs/reference/glossary/app-container.md:7]: I take reference from Korean and cncf glossary localization. They also used as same like this.

Comment (taniaduggal) [reply content/hi/docs/reference/glossary/app-container.md:7]: What do you think @divya-mohan0209

**Comment by taniaduggal** [comment]
@divya-mohan0209 Please take a look. I made suggested changes

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: content/hi/docs/reference/glossary/app-container.md (hunk 1 above)
Comment (anubha-v-ardhan) [reply content/hi/docs/reference/glossary/app-container.md:7]: @taniaduggal I would suggest:
```suggestion
  कार्यभार का एक हिस्सा चलाने के लिए प्रयोग किया जाने वाला कंटेनर। इनिट (init) कंटेनर के साथ तुलना करें।
```

File: content/hi/docs/reference/glossary/app-container.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ id: app-container
 date: 2019-02-12
 full_link:
 short_description: >
-  एक कंटेनर एक कार्यभार का हिस्सा चलाने के लिए प्रयोग किया जाता है। प्रारंभिक कंटेनर की तुलना में।
+  कार्यभार का एक हिस्सा चलाने के लिए प्रयोग किया जाने वाला कंटेनर। इनिट (init) कंटेनर के साथ तुलना करें।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/app-container.md:7]: I understand that this is just a reference doc. However can we start off by saying, @anubha-v-ardhan ?
```suggestion
ऐप कंटेनर कार्यभार का एक हिस्सा चलाने के लिए प्रयोग किया जाने वाला कंटेनर है। इनिट (init) कंटेनर के साथ तुलना करें।
```

Comment (Babapool) [reply content/hi/docs/reference/glossary/app-container.md:7]: Can we write something along the lines of?:
```suggestion
 एक ऐसा कंटेनर जो कार्यभार के एक हिस्से को चलने के प्रयोग में किया जाता हैं। इनिट (init) कंटेनर के साथ तुलना करें।
```
WDYT, Thoughts? Thanks.

Comment (anubha-v-ardhan) [reply content/hi/docs/reference/glossary/app-container.md:7]: I think @divya-mohan0209's suggestion `ऐप कंटेनर कार्यभार का एक हिस्सा चलाने के लिए प्रयोग किया जाने वाला कंटेनर है` sounds simpler.

**Pull request closed without merging**